### PR TITLE
Fix ci

### PR DESCRIPTION
**Pull request by yurikoles**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,20 @@
 language: cpp
+os: osx
+osx_image:
+  - xcode10.1
+  - xcode11.2
+env:
+  global:
+  - MAKEFLAGS="-j2"
+  - Qt5Core_DIR=/usr/local/opt/qt/lib/cmake/Qt5Core
```
Comment (tsdgeos) [inline .travis.yml:8]: never needed this, why would i need it now?

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +1,20 @@
 language: cpp
+os: osx
+osx_image:
+  - xcode10.1
+  - xcode11.2
+env:
+  global:
+  - MAKEFLAGS="-j2"
```
Comment (tsdgeos) [inline .travis.yml]: I'd prefer not to have this either, if you want to compile it faster install ninja and use that i guess

File: .travis.yml (hunk 1 above)
Comment (yurikoles) [reply .travis.yml:8]: Because I see in logs that poppler can't find a Qt

File: .travis.yml (hunk 2 above)
Comment (yurikoles) [reply .travis.yml]: Ok, I just checked the Travis CI docs and saw that every instance has 2 cores.

File: .travis.yml (hunk 1 above)
Comment (yurikoles) [reply .travis.yml:8]: So poppler-qt isn't built without this.

Comment (yurikoles) [reply .travis.yml:8]: My assumption is that building poppler-qt is in scope of CI of KDE dev :)

Comment (tsdgeos) [reply .travis.yml:8]: this is not really "KDE dev" more "poppler dev", but sure, builing most of the code is wanted.

**Comment by tsdgeos** [comment]
thanks :)

**Comment by yurikoles** [comment]
You helped me, I helped you, we are in one team :)

**Pull request merged**